src/PantherTestCaseTrait.php

```diff
@@ -29,32 +29,34 @@
     /**
      * @var bool
      */
-    public static $stopServerOnTeardown = true;
+    public static $stopServersOnTeardown = true;
 
     /**
      * @var string|null
      */
-    protected static $webServerDir;
+    protected static $webServerDir = null;
 
     /**
-     * @var WebServerManager|null
+     * @var WebServerManager[]
      */
-    protected static $webServerManager;
+    protected static $webServerManagers = [];
 
     /**
+     * The last started web server base uri.
+     *
      * @var string|null
      */
     protected static $baseUri;
```

Comment:
This property is a problem. If people relies on it, we would maybe need a method to get the "baseUri" depending of options. Another way would be to name the client / web server when you create them.